### ISSUE TITLE
Screen Flash Maybe Caused by HDR

### ISSUE THREAD

**Issue by AlexyGalaxy** [kolton/d2bot-with-kolbot#3081]
TLDR; HDR causes windows 10 to restart something making my entire screen go black for 2-3 secs when a bot crashes or is stopped. I still dont know whats happening.

When any of my bots stops or crashes my screen goes black for 2-3secs, only the game client that was crashing/closing disappears, and a new client opens to run the bot. Has anyone seen this issue or know a fix? I want HDR on because (I thought) my computer can handle it and the screen looks bad without it. This only started happening when I bought a new monitor with HDR so I'm guessing that's what did it.
There are no unusual errors on the console just normal ones like chestmaina.

My monitor is an ASUS TUF Gaming vg289 I'm on 64bit windows 10

Any help would be great thanks!

Edit: I turned off HDR and the issue went away... now if only I could find a fix to why hdr is causing this.

**Comment by DarkHorseDre** [comment]
I have no idea

if you start the game without the bot and enable HDR, does the same happen?
if so, its the game..
